styles.css

```diff
@@ -116,13 +119,12 @@ body{
     flex-direction: column;
     padding: 1.6rem;
 }
+
 .drawer__heading {
     margin: 0rem 0rem 1.6rem 0rem;
 }
 
-.drawer__title {
-
-}
+.drawer__title {}
```

Comment:
can be removed if not in use

Reply:
No, that's a placeholder for future use.